Delcam.ProductInterface.PowerMILL/Classes/PMProject.cs

```diff
@@ -423,6 +423,36 @@ public void CreateBlock(string modelName, double blockExpansion)
             _powerMILL.DoCommand("BLOCK ACCEPT");
         }
 
+        /// <summary>
+        /// This operation creates a Block in PowerMILL from an boundary. The heights are calculated automatically 
+        /// </summary>
+        /// <param name="boundary">The boundary from which to create the block.</param>        
+        public void CreateBlock(PMBoundary boundary)
```

Comment:
Try to avoid overloading.  We have found over time that you can end up with a lot of functions with the same name and it isn't clear what they are doing different to each other.  CreateBlockFromBoundary would be good here.  CreateBlockFromBoundaryWithLimits for the other.